### PR TITLE
Step description for p:tee

### PR DESCRIPTION
**Pull request by eriksiegel**
Covers #293

### CODE REVIEW

File: steps/src/main/xml/steps/tee.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+if the URI scheme is not supported or the step cannot store to the
+specified location.</error></para>
+
+<para>The output of this step is what appears on its input port, unchanged.</para>
+
+<para>The <option>serialization</option> option is provided to control the
+serialization of content when it is stored. Serialization is described in
+<biblioref linkend="xproc30"/>.</para>
+  
+  <para>If the <option>enabled</option> option is false, no attempt to store the document will be done. The step will
```
Comment (gimsieke) [inline steps/src/main/xml/steps/tee.xml]: will be made

File: steps/src/main/xml/steps/tee.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <p:output port="result" sequence="true" content-types="*/*"/>
+    <p:option name="href" required="true" as="xs:anyURI"/>        
+    <p:option name="serialization" as="map(xs:QName,item()*)"/>
+    <p:option name="enable" as="xs:boolean" select="true()"/>   
+  </p:declare-step>
+
+<para>The value of the <option>href</option> option
+<rfc2119>must</rfc2119> be an <type>anyURI</type>. If it is relative,
+it is made absolute against the base URI of the element on which it is
+specified (<tag>p:with-option</tag> or <tag>p:store</tag> in the case
```
Comment (gimsieke) [inline steps/src/main/xml/steps/tee.xml]: `p:store`?

Comment (eriksiegel) [reply steps/src/main/xml/steps/tee.xml]: must be p:tee

File: steps/src/main/xml/steps/tee.xml (hunk 1 above)
Comment (eriksiegel) [reply steps/src/main/xml/steps/tee.xml]: ok

**Comment by eriksiegel** [comment]
@xml-project You're right about the application of #534 on non-required options, but in this case it only applies to `serialization` . The `enabled` option has an explicit `@select`. I'll change it.

**Pull request merged**